storage/postgres/client.go

```diff
@@ -93,7 +94,7 @@ func (c *Client) SendBatch(ctx context.Context, batch *storage.QueryBatch) error
 		defer common.CloseOrLog(batchResults, c.logger)
 		for i := 0; i < pgxBatch.Len(); i++ {
 			if _, err := batchResults.Exec(); err != nil {
-				return err
+				return fmt.Errorf("query %d: %w", i, err)
```

Comment:
If you want, you could log `batch.Queries()[i]`. On one hand, too verbose to be the actual err message, but OTOH very helpful when we have batches of 200 queries and the error message says effectively "refer to query 132 in the above huge list"

Reply:
[i] would be good

Reply:
formatting with %v